Sources/FuzzilliCli/main.swift

```diff
@@ -30,6 +30,9 @@ Options:
                                   Available profiles: \(profiles.keys).
     --jobs=n                    : Total number of fuzzing jobs. This will start one master thread and n-1 worker threads. Experimental!
     --engine=name               : The fuzzing engine to use. Available engines: "mutation" (default), "hybrid", "multi"
+    --corpus=name               : The corpus scheduler to use. Available schedulers: "basic" (default), "markov"
+    --deterministicCorpus       : If set, only deterministic samples will be included in the corpus, up to a threshold
+    --maxResetCount=n           : The number of items to reset a non-deterministic edge before ignore it in subsequent executions
```

Comment:
Nit: `The number of times a non-deterministic edge is reset before it is ignored in subsequent executions`. Also, is this only used with `--deterministicCorpus`?

Reply:
It is only used with `--deterministicCorpus`. I'll add a check that exits if any of `--minDeterminismExecs`, `--maxDeterminismExecs`, or `--maxResetCount` are used without `--deterministicCorpus`